tesseract/tesseract_environment/CMakeLists.txt

```diff
@@ -96,3 +99,7 @@ if (TESSERACT_ENABLE_TESTING)
   add_run_tests_target(ENABLE ${TESSERACT_ENABLE_RUN_TESTING})
   add_subdirectory(test)
 endif()
+
+if (TESSERACT_ENABLE_BENCHMARKING)
+  add_subdirectory(test/benchmarks)
+endif()
```

Comment:
How do you add them to the CI?

Reply:
Well I guess we are running them. We just aren't displaying them. In the after script we need to copy the results like we are doing for collision. Then we'd have to modify the json to append them or something so that both show up on the ghpages